scaffold/package.json

```diff
@@ -17,13 +17,16 @@
   "author": "",
   "license": "UNLICENSED",
   "dependencies": {
-    "carbon-react": "^4.0.0",
+    "carbon-react": "^8.0.1",
+    "carbon-state-management": "^1.0.0",
+    "core-js": "^3.1.4",
     "flux": "^3.0.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "i18n-js": "http://github.com/fnando/i18n-js/archive/v3.0.0.rc12.tar.gz",
```

Comment:
Looking at [the NPM documentation for `package.json`](https://docs.npmjs.com/files/package.json#github-urls), I think this can be simplified to:
```
"i18n-js": "fnando/i18n-js#v3.0.0.rc12",
```